python/sparkdl/graph/utils.py

```diff
@@ -89,77 +87,81 @@ def get_tensor(graph, tfobj_or_name):
         name = tfobj_or_name.name
     if not isinstance(name, six.string_types):
         raise TypeError('invalid tensor request for {} of {}'.format(name, type(name)))
-    _tensor_name = as_tensor_name(name)
+    _tensor_name = tensor_name(name, graph=None)
     tnsr = graph.get_tensor_by_name(_tensor_name)
     assert tnsr is not None, \
         'cannot locate tensor {} in current graph'.format(_tensor_name)
     return tnsr
 
-def as_tensor_name(name):
+def tensor_name(tfobj_or_name, graph=None):
     """
     Derive tf.Tensor name from an op/tensor name.
-    We do not check if the tensor exist (as no graph parameter is passed in).
+    If the input is a name, we do not check if the tensor exist
```

Comment:
"If the input is a TensorFlow object, or the graph is given, we also check that the tensor exists in the associated graph."